### DISCUSSION TITLE
Minimum age: 16, 18, 21?

### DISCUSSION THREAD

**Discussion by Nonononoki** [Alovoa/alovoa#102] [General]
Moving https://github.com/Alovoa/alovoa/issues/56 to discussions

**Comment by zecamigo** [comment]
While agreeing 16 year olds should definitely be allowed to date, the trust and safety framework of the platform is still in its infancy, you should consider pushing it up to 18.

People will sign up with fake ages to groom minors even in larger platforms with ID verification processes.

Sixteen years old could also fake their age, but that will less likely leave admins in a though situation.

Safety in a dating site is hard and there should be a lot more discussion around this.

**Comment by Leaced** [comment]
As long as people under 18 are allowed on this platform:

- changing standard "minimum age" to 18 or above (depending on the age of the user)
- making it impossible to set this below 18 for people over 18

Searching for minors on this platform should not be possible for adults and it should not be encouraged by the standard settings.

**Reply by das-g** [reply]
Let's think this through: Let's assume we have two persons, Alice and Bob, born three months apart, with Alice being the older one. With the limits you suggest, Alice should be able to find Bob until she turns 18, then shouldn't be able to find Bob anymore for three months until he, too, turns 18, and only then should she be able to find him again.

That doesn't seem like very desirable behavior of the platform in general nor in Bob's best interest, does it? If the platform is to allow minors and if one of the involved persons is a minor, a rule based on age _differences_ is probably more practical and common-sense-ish than one based on only whether the other person isn't also a minor. (That's also why _reasonable_ legislations have _some_ variant of a ["Romeo and Juliet law"](https://en.wikipedia.org/wiki/Romeo_and_Juliet_laws).)
